setup.cfg

```diff
@@ -1,44 +1,38 @@
 [metadata]
 name = stsci.stimage
-version = 0.2.4
 author = STScI
 author-email = help@stsci.edu
+license = BSD-3-Clause
 summary = Various image processing functions
-home-page = https://hsthelp.stsci.edu
+project_urls =
+    Tracker = https://github.com/spacetelescope/stsci.stimage/issues
+    Documentation = https://stscistimage.readthedocs.io/en/latest/
+    Source Code = https://github.com/spacetelescope/stsci.stimage
 classifier = 
 	Intended Audience :: Science/Research
 	License :: OSI Approved :: BSD License
 	Operating System :: OS Independent
 	Programming Language :: Python
 	Topic :: Scientific/Engineering :: Astronomy
 	Topic :: Software Development :: Libraries :: Python Modules
-requires-python = >=3
-requires-dist = 
-	numpy (>=1.16)
 
-[extension=stsci.stimage._stimage]
-sources = 
-	src/immatch/geomap.c
-	src/immatch/xyxymatch.c
-	src/immatch/lib/tolerance.c
-	src/immatch/lib/triangles.c
-	src/immatch/lib/triangles_vote.c
-	src/lib/error.c
-	src/lib/lintransform.c
-	src/lib/polynomial.c
-	src/lib/util.c
-	src/lib/xybbox.c
-	src/lib/xycoincide.c
-	src/lib/xysort.c
-	src/surface/cholesky.c
-	src/surface/fit.c
-	src/surface/surface.c
-	src/surface/vector.c
-	src_wrap/stimage_module.c
-	src_wrap/wrap_util.c
-	src_wrap/immatch/py_xyxymatch.c
-	src_wrap/immatch/py_geomap.c
-include_dirs = 
-	include
-	src_wrap
-	numpy
+[options]
+zip_safe = False
+python_requires = >=3.6
+setup_requires =
+    setuptools_scm
+install_requires =
+	numpy>=1.14
```

Comment:
Inconsistent indentation throughout this file.

Reply:
omg, how did `tab`s end up in this file?!

Reply:
Fixed.